pytest_hot_reloading/plugin.py

```diff
@@ -117,6 +120,16 @@ def apply_assertion_rewrite(self, ast_func, glb):
                         nodes.append(field)
             return ast_func
 
+        def stash(self, lineno=1, col_offset=0):
```

Comment:
This is effectively the same as the original function except we're deducting 1 from delta